TURING.md

```diff
@@ -0,0 +1,6 @@
+#Team Turing
+##Best Team Eva!!!!
+amansouri47
+kowtekd
+taylorr82
```

Comment:
Are you sure you want this guy in your group?

Reply:
is that about me?